module/VuFind/src/VuFind/Search/Base/Params.php

```diff
@@ -1912,16 +1913,31 @@ public function getOverrideQuery()
     /**
      * Return search query object.
      *
-     * @return \VuFindSearch\Query\AbstractQuery
+     * @return AbstractQuery
      */
-    public function getQuery()
+    public function getQuery(): AbstractQuery
```

Comment:
Is there a strong reason to add the return type here? It obvious is a good idea in the long term, but I'm concerned it might be a BC break if any classes are extending this method. Maybe better to omit for now and do in a separate dev-10.0 PR.

Reply:
Oops, right, reverted.